### DISCUSSION TITLE
torch_api.cpp:1:9: fatal error: 'torch/csrc/autograd/engine.h' file not found

### DISCUSSION THREAD

**Discussion by manjunathsc24** [sugarme/gotch#77] [Q&A]
Hi sugarme
when i am installing gotch

i am getting below error , could you please let me know how to solve , i have followed all the steps mentioned but still getting the below error. it will be great helpful for me if you able to solve this issue.

go get "github.com/sugarme/gotch"

github.com/sugarme/gotch/libtch
torch_api.cpp:1:9: fatal error: 'torch/csrc/autograd/engine.h' file not found

thank you
Manjunath

**Comment by sugarme** [comment]
@manjunathsc24 ,
I guess your ‘libtorch’ not in the PATH. 

Please describe your computer OS, install with CUDA or not? Detail how did you install gotch?
